README.md

```diff
@@ -0,0 +1,200 @@
+# Awesome Books
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>Awesome Books</b></h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+- [🖼️ Project Gallery](#gallery)
+
+<!-- PROJECT DESCRIPTION -->
+# 📖 Awesome Books<a name="about-project"></a>
+
+> This is the First JS Project of Microverse.
+
+Capstone Project (Module 1) is built as part of the 12-month Microverse Full-Stack Web Development Program. The mobile project was first applied in this project, followed by the desktop one. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+This capstone project is built and configured using the following tools:
+
+<details>
+  <summary>Languages</summary>
+  <ul>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://www.javascript.com/">JavaScript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Development Applications</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+    <li><a href="https://getbootstrap.com/">Bootstrap</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Git / GitHub</summary>
+  <ul>
+    <li><a href="https://git-scm.com/">Git</a></li>
+    <li><a href="https://www.github.com/">Github</a></li>
+    <li><a href="https://pages.github.com/">Github Pages</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Web Configuration Applications</summary>
+  <ul>
+    <li><a href="https://developer.chrome.com/docs/lighthouse/overview/">Lighthouse</a></li>
+    <li><a href="https://stylelint.io/">Stylelint</a></li>
+    <li><a href="https://webhint.io/">Webhint</a></li>
+    <li><a href="https://eslint.org/">ES Lint</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+### Key Features <a name="key-features"></a>
+
+The three key features used in my XXX project include: 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+The Live Demo of this Capstone Project is available now on GitHub Pages and Loom.
+
```

Comment:
- [ ] Since there is currently no live demo available you can remove this section :+1: